include/ringmesh/geomodel/geomodel_builder.h

```diff
@@ -158,6 +158,40 @@ namespace RINGMesh
     {
     public:
         explicit GeoModelBuilder( GeoModel2D& geomodel );
+
+        void build_surfaces_from_corners_and_lines();
+
+    private:
+        struct OrientedLine
```

Comment:
This private section is an example of what pimpl principle can remove. You don't have to change it, but it could be nice.